src/components/dashboard/FaceVerification/sdk/ZoomSDK.native.js

```diff
@@ -21,7 +21,7 @@ export const ZoomSDK = new class {
     const { sdk, logger } = this
 
     try {
-      await sdk.preload()
+      sdk.preload && (await sdk.preload())
```

Comment:
```suggestion
      await sdk.preload()
```

you're free to modify react-native-zoom package. so just add the stub for that method there

Reply:
@serdiukov-o-nordwhale that would probably require a bit of debugging, this is just a hotfix to avoid crashing. knowing we still don't have native support for zoom I don't think it's necessary to spend time updating and debugging the lib, I'd rather do it when we get to work on zoom for native